Header and footer update: User Journeys/ Navigation

jblair87 opened the pull request:
Refer to User Journey Google Doc for task breakdown. 
https://docs.google.com/document/d/1k-Jaz5ZvRrVNlZZkZJT71YdCLcjGRQWtHfEJX-xB8Cw/edit?usp=sharing

jblair87 committed:
updated nav and footer

pranavsid98 commented:
Hi @jblair87 Could you provide me access to the Google doc so that I can read up on stuff before reviewing

pranavsid98 commented on core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html:
```diff
@@ -41,21 +41,21 @@
         </ul>
       </li>
-      <li ng-show="navElementsVisibilityStatus.I18N_TOPNAV_DONATE">
+      <!--<li ng-show="navElementsVisibilityStatus.I18N_TOPNAV_DONATE">
```
Why are we commenting code out here? Shouldn't we be removing it if its not in use? /cc @seanlip

seanlip replied:
yes, please delete code that's not used.

pranavsid98 reviewed (commented):
One minor nit: Otherwise LGTM!

pranavsid98 commented:
@jblair87 By the way, some tests seem to be failing, could you check it?

jblair87 committed:
Header Footer Update Fix: User Journey/Navigation

Updated. I commented it out in case some one wanted to add it back in in the future.

jblair87 committed:
fix travis build error

fix travis build error on css

jblair87 committed:
Merge pull request #2 from jblair87/jblair87-patch-1

Header Footer Update Fix: User Journey/Navigation

kevinlee12 commented:
Hi @jblair87, it looks like there are still some issues with the tests, ptal!

seanlip commented:
Superseded by #4174.
